slack-gamebot/commands/taunt.rb

```diff
@@ -0,0 +1,24 @@
+module SlackGamebot
+  module Commands
+    class Taunt < SlackRubyBot::Commands::Base
+      def self.call(client, data, match)
+        taunter = ::User.find_create_or_update_by_slack_id!(client, data.user)
+        arguments = match['expression'] ? match['expression'].split.reject(&:blank?) : []
+        if arguments.length > 1
```

Comment:
Maybe this should support multiple people? Especially that we support doubles and such. It would mean looking up everyone, then joining them with an "and".